README.md

```diff
@@ -1,12 +1,18 @@
 # ParticleDA
 
+[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://team-raddish.github.io/ParticleDA.jl/stable/)
 [![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://team-raddish.github.io/ParticleDA.jl/dev/)
 [![Build Status](https://github.com/Team-RADDISH/ParticleDA.jl/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/Team-RADDISH/ParticleDA.jl/actions/workflows/ci.yml?query=branch%3Amaster)
 [![Coverage](https://codecov.io/gh/Team-RADDISH/ParticleDA.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/Team-RADDISH/ParticleDA.jl)
```

Comment:
Unrelated to this PR, I just realised this is wrong because our default branch is called `master`, not `main`.  I'd be happy with renaming it to `main`, but I guess we should be consistent in these badges.  For the rest, this looks good

Reply:
I'm also happy with renaming `master` &rightarrow; `main`. Looking for `master` references in code as well as updating README we'd need to update Actions workflows. We also have lots of usages of `master_rank` which I guess it would sort of make sense to also update to `main_rank` or similar (though technically this would be a breaking change as we expose `master_rank` as a filter parameter). For purposes of this PR I guess probably easiest to update Codecov badge to point to `master` and do a separate PR for updating references to `main` (+ rename branch)?